.travis.yml

```diff
@@ -3,10 +3,11 @@ language: node_js
 node_js:
   - "0.10"
   - "4"
+  - "5"
 
 matrix:
   allow_failures:
-    - node_js: "4"
```

Comment:
I asked @tgius to make this change. All the node 4 builds are passing regularly I believe and I think we should start supporting it a little more strongly. on-http used to fail on node 4 but they we fixed a couple weeks ago. I can bring this up in CC or the mailing list if you want.